ChatSample/ChatSample/wwwroot/index.html

```diff
@@ -28,6 +28,10 @@
 
             // Get the user name and store it to prepend to messages.
             var name = prompt('Enter your name:', '');
+            while (name.length > 15 || name.length < 3 || !(/^[a-zA-Z]*$/.test(name))) {
+                alert("Invalid name try again!");
```

Comment:
Should we tell the user as to why the name is considered invalid? Perhaps (between 3 and 15 alphabets only)